examples/lb3-application/package.json

```diff
@@ -36,27 +36,27 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@loopback/booter-lb3app": "^1.0.0",
-    "@loopback/boot": "^1.2.3",
-    "@loopback/context": "^1.12.0",
-    "@loopback/core": "^1.6.0",
-    "@loopback/repository": "^1.5.1",
-    "@loopback/rest": "^1.10.4",
-    "@loopback/rest-explorer": "^1.1.18",
+    "@loopback/booter-lb3app": "^1.1.1",
+    "@loopback/boot": "^1.2.5",
+    "@loopback/context": "^1.13.0",
+    "@loopback/core": "^1.6.2",
+    "@loopback/repository": "^1.5.3",
+    "@loopback/rest": "^1.11.0",
+    "@loopback/rest-explorer": "^1.1.20",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
     "debug": "^4.1.1",
-    "helmet": "^3.16.0",
+    "helmet": "^3.18.0",
     "loopback": "^3.25.1",
     "loopback-boot": "^3.3.0"
   },
   "devDependencies": {
-    "@loopback/build": "^1.5.0",
-    "@loopback/http-caching-proxy": "^1.0.15",
-    "@loopback/rest": "^1.10.4",
-    "@loopback/testlab": "^1.2.5",
+    "@loopback/build": "^1.5.2",
+    "@loopback/http-caching-proxy": "^1.0.17",
```

Comment:
Is this module used by your `lb3-application` example at all? I think it's not, could you please remove it?